packages/contracts/scripts/shared/constants/addresses.ts

```diff
@@ -0,0 +1,57 @@
+        whaleAddress: "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
+        tester: "0x89eae71b865a2a39cba62060ab1b40bbffae5b0d",
+        sablier: "0xA4fc358455Febe425536fd1878bE67FfDBDEC59a",
+        DAI: "0x6B175474E89094C44Da98b954EedeAC495271d0F",
+        USDC: "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+        USDT: "0xdAC17F958D2ee523a2206206994597C13D831ec7",
+        curve3CrvBasePool: "0xbEbc44782C7dB0a1A60Cb6fe97d0b483032FF1C7",
+        curve3CrvToken: "0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490",
+        curveFactory: "0x0959158b6040D32d04c301A72CBFD6b39E21c9AE",
+        curveWhaleAddress: "0xC2872ab688400940d5a6041599b3F7de20730d49",
+        daiWhaleAddress: "0x16463c0fdB6BA9618909F5b120ea1581618C1b9E",
+        sushiMultiSig: "0x9a8541Ddf3a932a9A922B607e9CF7301f1d47bD1",
+        UbqWhaleAddress: "0xa53A6fE2d8Ad977aD926C485343Ba39f32D3A3F6",
+        MasterChefAddress: "0x8fFCf9899738e4633A721904609ffCa0a2C44f3D",
+        MetaPoolAddress: "0x20955cb69ae1515962177d164dfc9522feef567e",
+        BondingAddress: "0x831e3674Abc73d7A3e9d8a9400AF2301c32cEF0C",
+        BondingV2Address: "0xdAC17F958D2ee523a2206206994597C13D831ec7", // FAKE ADDRESS, TO BE REPLACED AFTER V2 DEPLOYMENT
+        UbiquityAlgorithmicDollarManagerAddress: "0x4DA97a8b831C345dBe6d16FF7432DF2b7b776d98",
+        jarUSDCAddr: "0xEB801AB73E9A2A482aA48CaCA13B1954028F4c94",
+        jarYCRVLUSDaddr: "0x4fFe73Cf2EEf5E8C8E0E10160bCe440a029166D2",
+        strategyYearnUsdcV2: "0xEecEE2637c7328300846622c802B2a29e65f3919",
+        usdcWhaleAddress: "0x72A53cDBBcc1b9efa39c834A540550e23463AAcB",
+        pickleControllerAddr: "0x6847259b2B3A4c17e7c43C54409810aF48bA5210",
+    },
+    hardhat: {
+        deployer: "0xefC0e701A824943b469a694aC564Aa1efF7Ab7dd",
+        whaleAddress: "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
+        tester: "0x89eae71b865a2a39cba62060ab1b40bbffae5b0d",
+        sablier: "0xA4fc358455Febe425536fd1878bE67FfDBDEC59a",
+        DAI: "0x6B175474E89094C44Da98b954EedeAC495271d0F",
+        USDC: "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+        USDT: "0xdAC17F958D2ee523a2206206994597C13D831ec7",
+        curve3CrvBasePool: "0xbEbc44782C7dB0a1A60Cb6fe97d0b483032FF1C7",
+        curve3CrvToken: "0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490",
+        curveFactory: "0x0959158b6040D32d04c301A72CBFD6b39E21c9AE",
+        curveWhaleAddress: "0xC2872ab688400940d5a6041599b3F7de20730d49",
+        daiWhaleAddress: "0x16463c0fdB6BA9618909F5b120ea1581618C1b9E",
+        sushiMultiSig: "0x9a8541Ddf3a932a9A922B607e9CF7301f1d47bD1",
+        UbqWhaleAddress: "0xa53A6fE2d8Ad977aD926C485343Ba39f32D3A3F6",
+        MasterChefAddress: "0x8fFCf9899738e4633A721904609ffCa0a2C44f3D",
+        MetaPoolAddress: "0x20955cb69ae1515962177d164dfc9522feef567e",
+        BondingAddress: "0x831e3674Abc73d7A3e9d8a9400AF2301c32cEF0C",
+        BondingV2Address: "0xdAC17F958D2ee523a2206206994597C13D831ec7", // FAKE ADDRESS, TO BE REPLACED AFTER V2 DEPLOYMENT
```

Comment:
I said "these" in my previous comment because there's two instances of this.

Reply:
what do you mean by `two instances`? I replaced json files with this

Reply:
updated in https://github.com/ubiquity/ubiquity-dollar/pull/261/commits/ff143a74102d05eef30b641a43251d11f6fe9734

Reply:
I'm not suggesting to remove just those lines. I'm suggesting that we remove the entire file. Review my earlier comment https://github.com/ubiquity/ubiquity-dollar/pull/261#discussion_r997311448

Reply:
agree to remove that file.

Reply:
https://github.com/ubiquity/ubiquity-dollar/pull/261/commits/399a6457f309087721e3b88689ac8bd4fb3af2ed